### ISSUE TITLE
sweep_function_start_stop does not exist in V 23.010 (eco top light)?!

### ISSUE THREAD

**Issue by lein1013** [LavermanJJ/pysolarfocus#28]
It seems that for V 23.010 (eco top light) the `sweep_function_start_stop` does not exist

Home Assistant System log:
``` 
Logger: root
Source: /usr/local/lib/python3.10/site-packages/pysolarfocus/modbus_wrapper.py:60
First occurred: 22:10:37 (141 occurrences)
Last logged: 22:33:41

Modbus read error at address=33410: Exception Response(131, 3, IllegalAddress)
```

related code 
https://github.com/LavermanJJ/pysolarfocus/blob/2f67825606316d0cd23a019f25a9e289a786e13f/pysolarfocus/components/pellets_boiler.py#L26

```
if api_version.greater_or_equal(ApiVersions.V_22_090.value):
    holding_address=33400
....
if api_version.greater_or_equal(ApiVersions.V_22_090.value):
            self.sweep_function_start_stop = DataValue(address=10,register_type=RegisterTypes.Holding)
            self.sweep_function_extend = DataValue(address=11,register_type=RegisterTypes.Holding)
```

**Comment by lein1013** [comment]
@LavermanJJ 

I was trying to add EcoTop in my pysolarfocus fork (see https://github.com/lein1013/pysolarfocus/blob/RCv3.6.5-2/pysolarfocus/__init__.py)

when I add it to my fork using
`
  "requirements": ["git+https://github.com/lein1013/pysolarfocus.git@releases/tag/RCv3.6.5-2", "pymodbus==3.2.2"],
`

of the solarfocus HACs integration (my fork https://github.com/lein1013/home-assistant-solarfocus), I still get

``` 
Dieser Fehler wurde von einer benutzerdefinierten Integration verursacht

Logger: custom_components.solarfocus.config_flow
Source: custom_components/solarfocus/config_flow.py:185
Integration: Solarfocus (Dev lein1013) (documentation, issues)
First occurred: 10:36:46 (1 occurrences)
Last logged: 10:36:46

Unexpected exception
Traceback (most recent call last):
  File "/config/custom_components/solarfocus/config_flow.py", line 227, in async_step_user
    await validate_input(self.hass, user_input)
  File "/config/custom_components/solarfocus/config_flow.py", line 195, in validate_input
    client = Solarfocus(hass, data=data)
  File "/config/custom_components/solarfocus/config_flow.py", line 185, in __init__
    system=Systems(data[CONF_SOLARFOCUS_SYSTEM]).name,
  File "/usr/local/lib/python3.10/enum.py", line 385, in __call__
    return cls.__new__(cls, value)
  File "/usr/local/lib/python3.10/enum.py", line 710, in __new__
    raise ve_exc
ValueError: 'EcoTop' is not a valid Systems
```

I've also tried to use 
``` 
  "requirements": ["git+https://github.com/lein1013/pysolarfocus.git@main#pysolarfocus==3.6.9", "pymodbus==3.2.2"],
```

It seems that my HA does not install the correct pysolarfocus version!? I think it should also work without `#pysolarfocus==3.6.9`
On my running HA I cann't really verify which pysolarfocus is now used
Any idea what I'm missing?

**Comment by lein1013** [comment]
Meanwhile I think that in HA core something regarding the installation of custom component python packages is broken - there is a issue open on this topic
https://github.com/home-assistant/core/issues/84121

**Comment by lein1013** [comment]
minor update: 
I'm currently using my fork https://github.com/lein1013/home-assistant-solarfocus/tree/main (some enu updated, commeted some registers that seemed to cause a problem, door contact inverted)
and the python lib is installed correctly when I use:
```
  "requirements": ["git+https://github.com/lein1013/pysolarfocus.git@ecotop#pysolarfocus", "pymodbus==3.2.2"],
```

**Comment by LavermanJJ** [comment]
fixed with #29
